pix2tex/train.py

```diff
@@ -47,7 +47,7 @@ def save_models(e, step=0):
     try:
         for e in range(args.epoch, args.epochs):
             args.epoch = e
-            dset = tqdm(iter(dataloader))
+            dset = tqdm(iter(train_dataloader))
```

Comment:
Actually the `iter` call is not necessary anymore.

Reply:
Yes, you are right. See that information on some [websites ](https://www.programiz.com/python-programming/iterator) but never use it as knowledge.

for is automatically create an iterator then use next to read the next element until raise `StopIteration `, am I right?